backend/controllers/adminController.js

```diff
@@ -1,21 +1,32 @@
-const User = require("../models/user");
+const User = require('../models/user');
 
 //Update role of a user
 function updateRole(req, res) {
-    User.update({
-        Role: req.body.role
-    }, {
+  console.log(req.body);
+
+  if (req.body.NewRole == req.body.OldRole) {
+    res.status(200);
+  } else {
+    User.update(
+      {
+        Role: req.body.NewRole,
+      },
+      {
         where: {
-            Email: req.body.email
-        }
-    }).then(result => {
-        console.log("Role successfully updated !");
-        res.status(200).send("Role successfully updated !");
-    }).catch(err => {
-        res.status(500).send("Error:"+err);
-    })
+          AccountId: req.body.AccountId,
```

Comment:
Same as comment mentioning styling convention

Reply:
Tested the update function locally, currently not working properly due to line 16. The accountId of the user should be used to verify if the user has the rights to update roles as an admin. The email should be used in this case to properly update the role of a user.